cmd/goal/node.go

```diff
@@ -132,15 +132,15 @@ var catchupCmd = &cobra.Command{
 func catchpointCmdArgument(cmd *cobra.Command, args []string) error {
 	catchpointsCount := 0
 	for _, arg := range args {
-		_, _, err := ledger.ParseCatchpointLabel(arg)
+		_, _, err := common.ParseCatchpointLabel(arg)
```

Comment:
I feel like `common` not a good name, it is too generic and it is hard to say what is it when first seeing line like this. Maybe `ledgerTypes` or `ledgerCommon`, or `lgdata`, `lgtypes`, `lgcom`, etc.
Especially we already have `daemon/algod/api/server/common` that is fine inside server but it leaked out to goal and libgoal and... not really common there.

Reply:
I think that you have a good point; but this need to be further hashed.
Most of the structures/interfaces in /ledger/common are internal to `/ledger/...`; so I think that it makes sense to keep it as `/ledger/common`. however, for the very few interfaces/struct that need to be exported, we might want to place these into a separate, public, package; maybe `/ledger/ledgercore` ?

Reply:
note - the Golang internal packages aren't a good fit here since `/ledger/internal` isn't accessible by `/ledger/trackers`